docs/tutorial/where-to-store-app-data.md

```diff
@@ -0,0 +1,93 @@
+# Where to Store App Data
+
+This tutorial provides guidance on where to store user data properly depending
+on the app's type.
+
+## User data paths
+
+In Electron, locations of user data can be obtained and customized with
+`app.getPath` and `app.setPath` APIs, and there are 3 types of paths are used
+most commonly: `userData`, `localUserData` and `userCache`.
+
+### `userData`
+
+The `userData` path is where app should store persistent user data, such as
+config files and authentication information. Generally you should not write
+large files in this location as some environments may choose to synchronize
+files in this location to cloud, especially on Window.
+
+It by default points to:
+
+* `%APPDATA%/AppName` on Windows
+* `$XDG_CONFIG_HOME/AppName` or `~/.config/AppName` on Linux
+* `~/Library/Application Support/AppName` on macOS
+
+### `localUserData`
+
+On Linux and macOS `localUserData` is an alias to `userData`, on Windows it
+points to `%LOCALAPPDATA%/AppName` instead.
+
+On Windows, both `localUserData` and `userData` can be used to store persistent
+user data, the difference between them is, the `userData` can be synchronized to
+cloud, but `localUserData` can not.
+
+The convention is, if the user data takes lots of disk space, they should be
+written to `localUserData`, otherwise it can be written to `userData` instead.
```

Comment:
I would start with the Windows portion here because I think the distinction is the most important part.

```suggestion
On Windows, `localUserData` points to `%LOCALAPPDATA%/AppName`. Both
`localUserData` and `userData` can be used to store persistent user data.
The difference between them is that  `userData` can be synchronized to
the cloud, but `localUserData` cannot.

As a rule of thumb, if some user data takes lots of disk space, it should be written
to `localUserData`. Otherwise, it should instead be written to `userData`.

On Linux and macOS, `localUserData` has no distinctive purpose and is just an alias
to `userData`.

```

Reply:
> On Linux and macOS, `localUserData` **has no distinctive purpose** and is just an alias
to `userData`.

Let me know if this part is wrong. This is just how I interpreted the paragraph.